run/test/specs/input_validations/onboarding_incorrect_seed.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { androidIt, iosIt } from '../../../types/sessionIt';
+import * as onboarding from '../locators/onboarding';
+import { openAppOnPlatformSingleDevice, SupportedPlatformsType } from '../utils/open_app';
+import { localize } from '../../../localizer/i18n/localizedString';
+
+iosIt('Onboarding incorrect seed', onboardingIncorrectSeed);
+androidIt('Onboarding incorrect seed', onboardingIncorrectSeed);
+
+// in this case we use the word 'zork' which is not on the mnemonic word list
+const incorrectSeed = 'ruby bakery illness push rift reef nabbing bawled hope zork silk lobster hope'
+// the expected error is 'Some words are incorrect' which is represented by the following localized string
+const expectedError = localize('recoveryPasswordErrorMessageIncorrect').strip().toString();
+
+async function onboardingIncorrectSeed(platform: SupportedPlatformsType) {
+    const { device } = await openAppOnPlatformSingleDevice(platform);
+    await device.clickOnElementAll(new onboarding.AccountRestoreButton(device));
+    await device.inputText(incorrectSeed, new onboarding.SeedPhraseInput(device));
+    // Trigger the validation by pressing Continue
+    await device.clickOnElementAll(new onboarding.ContinueButton(device));
+    if (platform === 'ios') {
+        // Wait for an error message to come up - the actual text is currently not exposed to Appium on iOS
+        await device.waitForTextElementToBePresent(new onboarding.ErrorMessage(device).build());
```

Comment:
Add a TODO here to update test once iOS has fixed their accessibility label

Reply:
done in all 5 files